spec/requests/internal/organizations_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "internal/organizations", type: :request do
+  let(:admin) { create(:user, :super_admin) }
+
+  before do
+    create_list :organization, 5
+    sign_in(admin)
+  end
+
+  describe "GETS /internal/organizations" do
+    it "lists all organizations" do
+      get "/internal/organizations"
+      Organization.all.each do |o|
+        expect(response.body).to include(o.name)
```

Comment:
is not this 5 expectations, instead of having one where you can just do `Organization.map(&:name)` to [name1, 2, 3,4,5]

Reply:
I changed this to something like closer to your example.